File: evaluation-corpus/c/trivial-assignment/ta_omar.c
#include <stdio.h>

int is_even(int value) {
    if (value % 2 == 0) {
        return 1;
    } else {
        return 0;
    }
}

int main(void) {
    printf("%d\n", is_even(7));
    return 0;
}

File: evaluation-corpus/java/trivial-assignment/ta_omar.java
public class EvenOmar {

    public boolean isEven(int value) {
        if (value % 2 == 0) {
            return true;
        } else {
            return false;
        }
    }

    public static void main(String[] args) {
        EvenOmar checker = new EvenOmar();
        System.out.println(checker.isEven(7));
    }
}
